drivers/scheduler/k8s/specs/test-sharedv4/pod.yaml

```diff
@@ -45,6 +45,24 @@ spec:
           mountPath: /test-sharedv4-vol
         - name: local-vol
           mountPath: /local-vol
+      - name: sv4test-reader
+        image: portworx/sharedv4-test:torpedo
+        imagePullPolicy: Always
+        command: ["python", "/app/reader.py"]
+        # sleeping for 9 seconds from 3 * number of pods * seconds
+        args: ["--interval=9", "$(LOCAL_FILE)"]
+        env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: LOCAL_FILE
+            value: "/local-vol/$(MY_POD_NAME)"
```

Comment:
Should we read from the sharedv4 vol instead of the local-vol?

Reply:
the reader reads from local-vol to get per-pod counter.